README.md

```diff
@@ -229,6 +230,8 @@ Fides is created and sponsored by Ethyca: a developer tools company building the
 
 [pypi-image]: https://img.shields.io/pypi/v/fidesctl.svg
 [pypi-url]: https://pypi.python.org/pypi/fidesctl/
+[conda-image]: https://anaconda.org/ethyca/fidesctl/badges/version.svg
```

Comment:
Iffy on adding this now since we don't have any tagged releases yet. The badge only shows my test upload - https://anaconda.org/ethyca/fidesctl/badges/version.svg

Reply:
might as well, i'm probably going to push a patch version to trigger a proper build for this